commands/new_function.go

```diff
@@ -152,6 +152,11 @@ Download templates:
 		}
 
 		fileName = appendFile
+
+		newLineErr := addLastNewline("./" + fileName)
```

Comment:
I would inline this and use the word `err`

```
if err := ... ; err != nil {
    return err
}
```

Reply:
Thanks for the suggestion!

Reply:
Resolved in 3a2cd9c